test/verifiedTests/dataIntegration/testMetaboTools/testCalculateLODs.m

```diff
@@ -0,0 +1,53 @@
+% The COBRA Toolbox: testcalculateLODs
+%
+% Purpose:
+%     - tests the calculateLODs function
+%
+% Authors:
+%     Loic Marx, November 2018
+%     
+%
+   
+
+% Save the current path
+currentDir = pwd;
+
+% Initialize the test
+fileDir = fileparts(which('testcalculateLODs'));
+
+cd(fileDir);
+
+% Defining the inputs
+lod_ngmL = 1;
+
+theo_mass = [1, 2];
+
+% Calculate the reference value
+
+v_ref = 1e-3 * [1; 0.5];
+
+% Calculate the actual value by using the function
+
+calculateLOD = calculateLODs(theo_mass,lod_ngmL);
+
+% Comparison between calculate value and ref value
+
+assert(isequal(calculateLOD, v_ref))
+
+% Throwing an error if the dimensions of both inputs are not equal
+lod_ngmL_unequal = [1, 2, 3];
+
+assert(verifyCobraFunctionError('calculateLODs', 'inputs', {theo_mass,lod_ngmL_unequal'}, 'testMessage', 'both inputs do not have the same size'));
```

Comment:
our tested message is not the same as the error message thrown. 
This should be  `'The length of the inputs are not the same'`, and I would actually suggest to improve that message:
`'The number of elements in the input vectors do not match. They have to be either the same size, or lod_ngmL has to be a single value which is used for all elements'`